### ISSUE TITLE
Feature request: Add support for Outbound Structured Field 'Load Programmed Symbols'

### ISSUE THREAD

**Issue by blackbit42** [pmattes/x3270#15]
This would enable custom fonts and raster graphics.

**Comment by pmattes** [comment]
I agree that Load Programmed Symbols support is probably the easiest route to supporting raster graphics, with custom font support coming along for free.

But it is still pretty far down on the to-do list. :-(
